extractor.go

```diff
@@ -615,14 +615,7 @@ func (this *contentExtractor) postCleanup(targetNode *goquery.Selection) *goquer
 			}
 
 			subParagraph := s.Find("p")
-			subParagraph.Each(func(j int, e *goquery.Selection) {
```

Comment:
This part is too aggressive, it strips paragraph titles etc.   See Test_BlogSpotCoUK()